chain-impl-mockchain/src/testing/scenario/template/mod.rs

```diff
@@ -235,6 +239,10 @@ impl From<VotePlanDef> for VotePlan {
             proposals,
             dto.payload_type,
             dto.committee_keys,
+            TokenIdentifier {
```

Comment:
probably we should add this field to `VotePlanDef`

Reply:
I did this, not sure if your approve still holds.